stylesheetFinal.css

```diff
@@ -296,35 +284,17 @@ input[id="email"] {
 
     .send-button {
         position: relative;
-        /* left: calc(50% - 75px); */
         left: 0;
         margin: 0;
     }
 }
 
 @media screen and (max-width: 500px) {
-    textarea {
-        width: 77.8vw;
-    }
-
-    /* fieldset {
-        min-width: none;
-        width: 77.8vw;
-        background-color: blue;
-    } */
-
-    input[type="radio"] {       
-        margin-right: calc(2.5vw);
+    h1 {
+        letter-spacing: 1.6vw;
```

Comment:
It's not realy common to use vw for letter spacign. It works fine in this case, but you should probably just use smaller fonts on mobile